### PR TITLE
Add toolbox pmrread

### PR DESCRIPTION
**Pull request by llacher**

### CODE REVIEW

**Comment by stschiff** [comment]
Thanks, @llacher and @TCLamnidis, but when I try to preview it says:

```
stephan_schiffels@biomac04 comp_human_adna_book % quarto preview
Preparing to preview
[1/1] pmrread.qmd
File pmrread.bib not found in resource path
```

Could you please add the bib-file?

**Review by stschiff — APPROVED** [review]
Hi @martynamolak and @TCLamnidis. This looks great. I do have a number of small comments about formulations and English here and there, that I'd like to put forward as a separate PR, but I think it would be good to merge this now, as I think it's basically ready, and I will then perhaps write an editing-PR later and ask you for review.

**Comment by stschiff** [comment]
Ah, sorry, but the missing bib-file needs to be resolved, of course, before we can merge this. Thanks.

**Comment by stschiff** [comment]
Actually I just realise that of course we don't want individual bib-files. Please just add your references to the existing `references.bib`. Thanks.

**Comment by llacher** [comment]
@stschiff I already added the references to the existing .bib file. Are they missing?

File: pmrread.qmd [hunk 1]
```diff
@@ -0,0 +1,318 @@
+---
+title: "Pairwise mismatch rate (PMR) and READ relatedness inference"
+author: "Martyna Molak"
+date: "02/15/2024"
+editor: visual
+bibliography: pmrread.bib
```
Comment (TCLamnidis) [inline pmrread.qmd:6]:
```suggestion
bibliography: references.bib
```

Comment (stschiff) [reply pmrread.qmd:6]: Yes, indeed, that's the one.

**Comment by martynamolak** [comment]
It's all good now for the pmr chapter publication or should I still do something?

**Pull request merged**